core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -3315,10 +3315,14 @@ payment.payid.info=A PayID like a phone number, email address or an Australian B
   bank, credit union or building society account. You need to have already created a PayID with your Australian financial institution. \
   Both sending and receiving financial institutions must support PayID. For more information please check [HYPERLINK:https://payid.com.au/faqs/]
 payment.amazonGiftCard.info=To pay with Amazon eGift Card you need to purchase an Amazon eGift Card at your Amazon account and \
-  use the BTC seller''s email or mobile nr. as receiver. Amazon sends then an email or text message to the receiver. \
-  Use the trade ID for the message field.\n\n\
-  Amazon eGift Cards can only be redeemed by Amazon accounts with the same currency.\n\n\
-  For more information visit the Amazon eGift Card webpage. [HYPERLINK:https://www.amazon.com/Amazon-1_US_Email-eGift-Card/dp/B004LLIKVU]
+  use the BTC seller''s email or mobile nr. as receiver. Amazon sends then an email or text message to the receiver.\n\
+  Amazon eGift Cards can only be redeemed by Amazon accounts with the same currency.\n\
+  For more information visit the Amazon eGift Card webpage. [HYPERLINK:https://www.amazon.com/Amazon-1_US_Email-eGift-Card/dp/B004LLIKVU]\n\n\
+  It is recommended to not send more than 100 USD per gift card as it seems that Amazon is considering higher amounts as \
+  risky and often flags such transactions as fraudulent.\n\
+  Split the amount up to several smaller amounts and use some creative reference text instead of the trade ID (e.g. 'Happy birthday Susan').\n\
+  Exchange in the trade chat with the peer which reference text you have used so they are able to \
+  assign the payment to the trade.
```

Comment:
I think it would be good if @m52go could have a look at this text to make it easier to understand. Might help avoid some mediation cases.

Reply:
Agree!